infra/resources/dev/locals.tf

```
@@ -1,12 +1,12 @@
 locals {
-  env_short = "p"
+  env_short = "d"
   location  = "westeurope"
 
   tags = {
     CostCenter  = "TS310 - PAGAMENTI & SERVIZI"
```

Shall we use the Engineering cost center? cc @Krusty93 
```suggestion
    CostCenter  = "TS700 - ENGINEERING"
```

---

Discard this suggestion. As of now, we don't have certainity about the correctness of this costcenter